src/TransformRunner.ts

```diff
@@ -47,38 +45,6 @@ export default class TransformRunner {
     return transform(source.content, {
       filename: source.path,
       babelrc: false,
-      parserOpts: {
```

Comment:
All this stuff moved into `RecastPlugin` since parser and generator options are now handled by plugin hooks.